### PR TITLE
Refactor layouts for flexibility

### PR DESCRIPTION
**Pull request by meyerhp**
### Description
This PR
-  Contains all the changes we made to support the Chronicle layout. It mainly breaks the theme down into partials so we can override specific sections from a site level without having to duplicate layouts.
- Fixes the duplicate title bug for single-page articles

### Issue
https://spandigital.atlassian.net/browse/PRSDM-3637
https://spandigital.atlassian.net/browse/PRSDM-3623

### CODE REVIEW

**Review by christopherbrunsdon — APPROVED** [review]
Looks good on my local environment

File: assets/_sass/_structure.scss [hunk 1]
```diff
@@ -410,3 +410,17 @@
...
+  }
+
+  p.archive-item {
+    font-size: 16px;
+    padding: 2px 0;
+    display: grid;
+    grid-template-columns: 1fr 9fr;
+    grid-gap: 32px;
+  }
+}
```
Comment (CharlRitterCompany) [inline assets/_sass/_structure.scss:426]: newline

File: layouts/index.menuindex.json [hunk 2]
```diff
@@ -1 +1 @@
-{{ (partial "json-navigation" .) | jsonify }}
\ No newline at end of file
+{{ (partial "json/root" .) | jsonify }}
```
Comment (CharlRitterCompany) [inline layouts/index.menuindex.json:1]: newline

File: layouts/partials/article/content.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+<article>{{ .Content }}</article>
```
Comment (CharlRitterCompany) [inline layouts/partials/article/content.html:2]: newline

File: layouts/index.searchmap.json [hunk 4]
```diff
@@ -1 +1 @@
-{{ (partial "searchmap-root" .) | jsonify }}
\ No newline at end of file
+{{ (partial "searchmap/root" .) | jsonify }}
```
Comment (CharlRitterCompany) [inline layouts/index.searchmap.json:1]: newline

File: layouts/partials/article/frontmatter.html [hunk 5]
```diff
@@ -22,8 +22,8 @@
...
 {{ if or $isRoles $isAuthor $isStatus }}
     <div class="article-header">
-        {{ partial "author" . }}
-        {{ partial "status" . }}
-        {{ partial "roles" . }}
+        {{ partial "article/author" . }}
+        {{ partial "article/status" . }}
+        {{ partial "article/roles" . }}
     </div>
 {{ end }}
```
Comment (CharlRitterCompany) [inline layouts/partials/article/frontmatter.html:29]: newline

File: layouts/partials/article/title.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+            <h1> {{ .Title }}</h1>
+        {{ else }}
+            <h2> {{ .Title }}</h2>
+        {{ end }}
+                
+        <div class="permalink">
+            <a style="cursor:pointer;" onclick="copyPermalink( {{ $articleLink }} )" id="{{.Page.RelPermalink}}" data-slug="#{{ $slug }}" class="link-icon" title="Permalink to this article"></a>
+        </div>
+    </div>
+{{ end }}
```
Comment (CharlRitterCompany) [inline layouts/partials/article/title.html:18]: newline

File: layouts/partials/common/pages.html [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{{/*  Returns the pages after filtering out the archived articles  */}}
+{{ $pages := .Data.Pages }}
+{{ if and $.Site.Params.archive (ne $.Site.Params.archive.age 0)}}
+    {{ $age := mul 86400 $.Site.Params.archive.age }}
+    {{ $before := sub now.Unix $age }}
+    {{ $pages = where .Data.Pages "Date.Unix" "gt" $before }}
+{{ end }}
+{{ return $pages }}
```
Comment (CharlRitterCompany) [inline layouts/partials/common/pages.html:8]: newline

File: layouts/partials/common/slug.html [hunk 8]
```diff
@@ -0,0 +1,8 @@
+{{/* Use title if no slug */}}
+{{ $slug := humanize .Params.Title }} {{/* Handles any dashes in the title */}}
+{{ $slug = anchorize $slug }}
+{{ $slug = replaceRE `(-{2,})` "-" $slug }} {{/* Replaces multiple dashes with one */}}
+{{ if .Params.Slug }}
+    {{ $slug = .Params.Slug}}
+{{ end }}
+{{ return $slug }}
```
Comment (CharlRitterCompany) [inline layouts/partials/common/slug.html:8]: newline

File: layouts/partials/page/header.html [hunk 9]
```diff
@@ -0,0 +1,8 @@
+<h1 class="page-title offendingHeading">
+{{ if eq (.Scratch.Get "scope") "single" }}
+    {{ .Parent.Title }}
+{{ else }}
+    {{ .Title}}
+{{ end }}
+</h1>
+<hr class="offendingHr" />
```
Comment (CharlRitterCompany) [inline layouts/partials/page/header.html:8]: newline

File: layouts/partials/page/list.html [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+                {{ $pageScopes := .Params.scope }}
+                {{ $isRendering = gt (len (intersect $pageScopes $siteScopes)) 0 }}    
+            {{ end }}
+        {{ end }}
+
+        {{ if $isRendering }}
+            {{ partial "article/root" . }}
+        {{ end }}
+    {{ end }}
+{{ end }}
```
Comment (CharlRitterCompany) [inline layouts/partials/page/list.html:39]: newline

File: layouts/partials/page/script.html [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+          linkContainer.appendChild(link);
+          heading.insertAdjacentElement("beforebegin", container);
+          container.appendChild(heading);
+          container.appendChild(linkContainer)
+        }
+      })
+    });
+  })();
+</script>
+{{ end }}
```
Comment (CharlRitterCompany) [inline layouts/partials/page/script.html:112]: newline

File: layouts/partials/searchmap/root.html [hunk 12]
```diff
@@ -4,7 +4,7 @@
 {{ range $menu := .Site.Menus.main.ByWeight }}
     {{ $urilzeIdentifier = $menu.Identifier | urlize  }}
     {{ with $.Site.GetPage "section" $urilzeIdentifier }}
-        {{ $items = $items | append (partial "searchmap-item" (dict "Page" . "Level" 1) )}}
+        {{ $items = $items | append (partial "searchmap/item" (dict "Page" . "Level" 1) )}}
     {{ end }}
 {{ end }}
 {{ return $items }}
```
Comment (CharlRitterCompany) [inline layouts/partials/searchmap/root.html:10]: newline

**Review by CharlRitterCompany — APPROVED** [review]
LGTM, just minor nit pick on newlines at end of files

**Review by CharlRitterCompany — APPROVED** [review]

**Pull request merged**